prime-dotnet-webapi/Controllers/HealthAuthoritySitesController.cs

```diff
@@ -22,15 +22,18 @@ public class HealthAuthoritySitesController : PrimeControllerBase
         private readonly IHealthAuthoritySiteService _healthAuthoritySiteService;
         private readonly IHealthAuthorityService _healthAuthorityService;
         private readonly ISiteService _siteService;
+        private readonly IEmailService _emailService;
 
         public HealthAuthoritySitesController(
             IHealthAuthoritySiteService healthAuthoritySiteService,
             IHealthAuthorityService healthAuthorityService,
-            ISiteService siteService)
+            ISiteService siteService,
+            IEmailService emailService)
```

Comment:
we alphabatize our DI now, could you please add this new service and the associated local variable at the top of the list?

Reply:
Alphabetized